contrib/pcap2emulation.py

```diff
@@ -662,22 +667,33 @@ def parse_file(self, file: str) -> None:
                                         )
                                         interface["ExtraData"] = d
                                         break
+                                    else:
+                                        sys.stderr.write(
+                                            "Unknown interface class: "
+                                            + str(interface["InterfaceClass"])
+                                        )
+                                        exit(1)
+                            else:
+                                sys.stderr.write(
+                                    "Unknown descriptor type: " + descriptor_type
+                                )
+                                exit(1)
 
                     elif (
                         "usb_usb_bmRequestType_type" in layers
-                        and int(layers["usb_usb_bmRequestType_type"], 16) == URB_CONTROL
+                        and get_int(layers["usb_usb_bmRequestType_type"]) == URB_CONTROL
                     ):
                         # Found vendor CONTROL URB request
                         direction = not (layers["usb_usb_bmRequestType_direction"])
                         s = f"ControlTransfer:Direction=0x{direction:02x}"
                         s += ",RequestType=0x{:02x}".format(
-                            int(layers["usb_usb_bmRequestType_type"], 16)
+                            get_int(layers["usb_usb_bmRequestType_type"])
                         )
                         s += ",Recipient=0x{:02x}".format(
-                            int(layers["usb_usb_bmRequestType_recipient"], 16)
+                            get_int(layers["usb_usb_bmRequestType_recipient"])
                         )
                         s += f",Request=0x{int(layers['usb_usb_setup_bRequest']):02x}"
-                        s += f",Value=0x{int(layers['usb_usb_setup_wValue'], 16):04x}"
+                        s += f",Value=0x{int(layers['usb_usb_setup_wValue']):04x}"
                         s += f",Idx=0x{int(layers['usb_usb_setup_wIndex']):04x}"
```

Comment:
Should those `int` be changed to `get_int`?

Reply:
Yeah that makes sense, thanks adjusted.